src/main/java/oncall/mapper/EmployeesMapper.java

```diff
@@ -0,0 +1,38 @@
+package oncall.mapper;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import oncall.domain.Employees;
+import oncall.util.ExceptionUtil;
+import oncall.util.StringUtil;
+
+public class EmployeesMapper {
+
+    private EmployeesMapper() {
+
+    }
+
+    public static Employees toEmployees(String input) {
+        List<String> separated = seperate(input);
+        validateDuplicated(separated);
+
+        return Employees.create(separated);
+    }
+
+    private static List<String> seperate(String input) {
+        String deleteSpaces = StringUtil.removeAllSpaces(input);
+        List<String> separated = Arrays.stream(deleteSpaces.split(","))
+                .toList();
+        return separated;
```

Comment:
이부분은 철저히 취향의 영역이라 생각합니다. 다만, 저의 경우 변수가 늘어날수록 의도하지 않았던 값의 변경 등으로 사이드 이팩트가 발생할 가능성이 높아진다 라고 생각하기에 필요없는 지역변수 역시 최소화 하려고 노력하고 있어요!
이 관점에서는 바로 `return` 될 지역변수라면 굳이 따로 변수로 추출하지 않는 것이 좋다고 생각합니다!

```suggestion
        String deleteSpaces = StringUtil.removeAllSpaces(input);
        return Arrays.stream(deleteSpaces.split(","))
                .toList();
```